_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -77,6 +77,14 @@ function page_render() {
 			return $this->render_nojs_configurable();
 		}
 		?>
+		<?php
+			/**
+			 * Fires when a notice is displayed in the Jetpack menu.
+			 *
+			 * @since 3.0.0
+			 */
+			do_action( 'jetpack_notices' );
```

Comment:
Since the action is already documented somewhere else, you can replace your doblock with this:

`/** This action is already documented in views/admin/admin-page.php */`

That makes things easier for our parser, and avoids having duplicate documentation.